tests/js/server/aql/aql-functions-numeric.js

```diff
@@ -5767,26 +5768,18 @@ function ahuacatlNumericFunctionsTestSuite () {
 ////////////////////////////////////////////////////////////////////////////////
 
     testDecayGauss : function () {
-        var data = [
+        [
           [1, 0, 10, 0, 0.2, 0.9840344433634576],
           [-10, 40, 5, 0, 0.1, 1.0000000000000458e-100],
           [41, 40, 5, 5, 0.7, 1]
-        ];
+        ].forEach(function (value) {
+          const [arg, origin, scale, offset, decay] = value;
```

Comment:
May be also unpack expected value? Therefore we can avoid using indexes in "value" variable while comparing real and expected values.

Reply:
Good point!